googletest/CMakeLists.txt

```diff
@@ -131,6 +139,12 @@ if(INSTALL_GTEST)
     @ONLY)
   install(FILES "${CMAKE_BINARY_DIR}/gtest.pc" "${CMAKE_BINARY_DIR}/gtest_main.pc"
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
+
+  # install CMake export files
+  install(EXPORT GtestTargets
+    FILE GtestConfig.cmake
+    NAMESPACE GTest::
+    DESTINATION cmake)
```

Comment:
`DESTINATION` should take `${CMAKE_INSTALL_LIBDIR}` into account. Therefore, change this line to:
`DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/Gtest`